From 594ea0ff2e28d43641b3e931bae15df8d85a953e Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Mon, 10 Oct 2016 16:25:54 +0800
Subject: [PATCH 1174/5242] MLK-13305 ARM: dts: Correct the setpoints table on
 imx6ull arm2 board

commit  8ed74419c8047c45541020e9cbd95cbc40feeef7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct the setpoints when LDO enabled on i.MX6ULL DDR3 arm2 board.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts
index c0fa279..e0dc29b 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-ldo.dts
@@ -12,12 +12,14 @@
 	operating-points = <
 		/* kHz	uV */
 		528000	1175000
-		99000	950000
+		396000	1025000
+		198000	950000
 	>;
 	fsl,soc-operating-points = <
 		/* KHz	uV */
 		528000	1175000
-		99000	1175000
+		396000	1175000
+		198000	1175000
 	>;
 	arm-supply = <&reg_arm>;
 	soc-supply = <&reg_soc>;
-- 
1.7.9.5

